BEST_PRACTICES.md

```diff
@@ -123,8 +136,8 @@ Image names should be named consistently to preserve organization and developer
 
 **For example:**
 
-* `refresh-bar-button-item` / `refresh-bar-button-item@2x` and `refresh-bar-button-item-selected` / `refresh-bar-button-item-selected@2x`
```

Comment:
Not anymore, because I've been pushing to reduce assets on all my projects for years. 😂 But for instance, the custom keyboard on iA Writer for iOS is all done in code. No assets there.